client/src/components/Workflow/Editor/Node.vue

```diff
@@ -43,6 +43,12 @@
                 <i class="fa fa-files-o" />
             </b-button>
             <i :class="iconClass" />
+            <span
+                class="node-title"
+                v-b-tooltip.hover
+                title="Index of the step in the workflow run form. Steps are ordered by distance to the upper-left corner of the window; inputs are listed first."
+                >{{ id }}:
```

Comment:
Starting numbering from 1 rather than 0 might be better from a UI point of view. Personally I prefer that the numbering matches that of the API, but I guess most users don't care about that.

Reply:
Uhh, that'll be an endless debate 🍿. I think I also prefer 0 ... but I know others prefer 1. Since we start hids at 1 I guess going 1-based is more consistent.